esphome/components/max7219digit/max7219digit.cpp

```diff
@@ -132,7 +132,7 @@ void HOT MAX7219Component::draw_absolute_pixel_internal(int x, int y, Color colo
     this->max_displaybuffer_.resize(x + 1, this->bckgrnd_);
   }
 
-  if ((y >= this->get_height_internal()) || (y < 0) || (x < 0))  // If pixel is outside display then dont draw
+  if (y >= this->get_height_internal() || y < 0)  // If pixel is outside display then dont draw
```

Comment:
if there is any reason to do this, please do in a different PR
```suggestion
  if ((y >= this->get_height_internal()) || (y < 0) || (x < 0))  // If pixel is outside display then dont draw
```